<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VRCWProjectionVposFloorClass</class>
 <widget class="QWidget" name="VRCWProjectionVposFloorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VRCWProjectionVposFloor</string>
  </property>
  <layout class="QVBoxLayout" name="VRCWProjectionVposFloorClassVLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item alignment="Qt::AlignLeft|Qt::AlignTop">
    <widget class="QWidget" name="mainWidget" native="true">
     <layout class="QGridLayout" name="mainWidgetGLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>15</number>
      </property>
      <item row="1" column="0" alignment="Qt::AlignLeft|Qt::AlignTop">
       <widget class="QLabel" name="pwCaveHintLabel">
        <property name="text">
         <string>Powerwall or Cave Hint</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignLeft|Qt::AlignTop">
       <widget class="QWidget" name="posWidget" native="true">
        <layout class="QGridLayout" name="posWidgetGLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>20</number>
         </property>
         <item row="0" column="0" alignment="Qt::AlignLeft|Qt::AlignTop">
          <widget class="QLabel" name="vPosLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Viewer position [mm]</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" alignment="Qt::AlignLeft|Qt::AlignTop">
          <widget class="QWidget" name="vPosWidget" native="true">
           <layout class="QHBoxLayout" name="vPosHLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="vPosXLabel">
              <property name="text">
               <string>x</string>
              </property>
              <property name="indent">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft|Qt::AlignTop">
             <widget class="QSpinBox" name="vPosXSpinBox">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="minimum">
               <number>-9999</number>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="vPosHSpacer1">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="vPosYLabel">
              <property name="text">
               <string>y</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft|Qt::AlignTop">
             <widget class="QSpinBox" name="vPosYSpinBox">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="minimum">
               <number>-9999</number>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
              <property name="value">
               <number>-1500</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="vPosHSpacer2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="vPosZLabel">
              <property name="text">
               <string>z</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft|Qt::AlignTop">
             <widget class="QSpinBox" name="vPosZSpinBox">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="minimum">
               <number>-9999</number>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
              <property name="value">
               <number>575</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0" alignment="Qt::AlignLeft|Qt::AlignTop">
          <widget class="QLabel" name="floorLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Floor height [mm]</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" alignment="Qt::AlignLeft|Qt::AlignTop">
          <widget class="QSpinBox" name="floorSpinBox">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <number>-9999</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="value">
            <number>-1125</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>vPosXSpinBox</tabstop>
  <tabstop>vPosYSpinBox</tabstop>
  <tabstop>vPosZSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
